src/components/ChecCard/InnerBlock.vue

```diff
@@ -75,7 +75,7 @@ export default {
 <style lang="scss">
 .card-inner-block {
   @apply
-    flex flex-row items-center justify-between w-full bg-gray-100 rounded p-4
+    flex-row items-center justify-between w-full bg-gray-100 rounded p-4
```

Comment:
```suggestion
    w-full bg-gray-100 rounded p-4
```

Theoretically all those don't matter if it's not `display: flex`.